### PR TITLE
update permissions to publish the Jira Cloud plugin

### PR DESCRIPTION
**Pull request by thombergs**
# Description

Updating permissions for the atlassian-jira-software-cloud plugin to add myself and remove two people who are no longer working on it.

I am already a committer in the plugin repository, see this PR: https://github.com/jenkinsci/atlassian-jira-software-cloud-plugin/pull/63. So I only need the additional upload permission so I can perform releases.

cc @bgvozdev and @lenin-atlassian as existing uploaders to approve the change.

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name (in my case, both are the same)
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

File: permissions/plugin-atlassian-jira-software-cloud.yml [hunk 1]
```diff
@@ -7,6 +7,5 @@ paths:
   - "com/atlassian/jira/software/cloud/jenkins/atlassian-jira-software-cloud"
 developers:
   - "lenin_atlassian"
-  - "vikrambollu_atlassian"
   - "bgvozdev"
-  - "nsurendran"
+  - "thombergs"
```
Comment (thombergs) [inline permissions/plugin-atlassian-jira-software-cloud.yml:11]: This is my GitHub username as well as my Jenkins LDAP user name.

The two people I removed are no longer with our team here at Atlassian and will no longer work on the plugin

**Comment by bgvozdev** [comment]
I approve

**Comment by timja** [comment]
The build in this repo is currently broken, see https://github.com/jenkins-infra/helpdesk/issues/2732 cc @dduportal

**Comment by bgvozdev** [comment]
@timja @dduportal Wondering if there's anything we could do to help resolving the issue and having this PR unblocked? 🙏

**Comment by halkeye** [comment]
I''ll re-run the job, there's been infra/ci issues recently and some failures

**Pull request merged**